config/desc/imsim/readout.py

```diff
@@ -12,9 +208,10 @@ def finalize(self, config, base, main_data, logger):
 
         This function will be called after all images have been built.
 
-        It returns some sort of final version of the object.  In the base class, it just returns
-        self.data, but depending on the meaning of the output object, something else might be
-        more appropriate.
+        It returns some sort of final version of the object.  In the
+        base class, it just returns self.data, but depending on the
+        meaning of the output object, something else might be more
+        appropriate.
```

Comment:
Could mention that your final version is an HDUList rather than a list of images.  This doc string was for the base class implementation.